extensions/amp-a4a/amp-ad-refresh.md

```diff
@@ -0,0 +1,65 @@
+<!---
```

Comment:
Not sure whether this is the appropriate location for this information; will move upon reviewer's suggestion.